build-system/tasks/storybook-react/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "amp-storybook",
+  "version": "0.1.0",
+  "description": "amp storybook tasks",
```

```suggestion
  "description": "amp-storybook tasks",
```

---

I'm going to put this as 
```suggestion
  "description": "storybook-react tasks to start bento components using their react binaries",
```
thinking it over it's not really for amp. It's more for bento.